Gems/AtomTressFX/Code/Rendering/HairRenderObject.cpp

```diff
@@ -922,23 +929,18 @@ namespace AZ
                         float DistanceRatio = AZStd::min((distance - MinLODDist) / AZStd::max(MaxLODDist - MinLODDist, 0.00001f), 1.f);
 
                         // Lerp: x + s(y-x)
-                        float MaxLODFiberRadius = FiberRadius * (shadowUpdate ? parameters->m_ShadowLODWidthMultiplier : parameters->m_LODWidthMultiplier);
-                        FiberRadius = FiberRadius + (DistanceRatio * (MaxLODFiberRadius - FiberRadius));
+                        float MaxLODFiberRadius = fiberRadius * (shadowUpdate ? parameters->m_ShadowLODWidthMultiplier : parameters->m_LODWidthMultiplier);
```

Comment:
nit: do a pass and rename parameters starting with capital to lower case (originated by AMD)

Reply:
nit: i think most of the value defined here could add a const